solutions/usaco-963.mdx

```diff
@@ -132,4 +134,54 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+```java
+import java.io.*;
+import java.util.*;
+
+public class CowGymnastics {
+	public static void main(String[] args) throws IOException {
+		BufferedReader r = new BufferedReader(new FileReader("gymnastics.in"));
+		PrintWriter pw = new PrintWriter(new FileWriter("gymnastics.out"));
+		StringTokenizer st = new StringTokenizer(r.readLine());
+
+		boolean[][] completePairs = new boolean[20][20];
+		int k = Integer.parseInt(st.nextToken());
+		int n = Integer.parseInt(st.nextToken());
+		int line[] = new int[n];
+		int count = 0;
+
+		while (k != 0) {
```

Comment:
kinda nitpicky but could you just use a for loop like
```java
for (; k > 0; k--) {
}
```

Reply:
or just
```java
for (int i = 0; i < k; i++) {
}
```

Reply:
uh really sorry
but could you change it to the second one
it's more intuitive

Reply:
no problem chief